content/20.treatments.md

```diff
@@ -472,13 +482,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
 
 ###### Current Evidence
 
-A recent preprint postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19.
-This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations [@doi:10.20944/preprints202003.0235.v2].
+A recent review postulated that an individual’s vitamin D status may significantly affect their risk of developing COVID-19.
+This hypothesis was derived from the fact that the current pandemic emerged in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low [@doi:10.3390/nu12040988].
 The authors suggest that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
 The authors also suggest supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff.
-While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically.
-Likewise, it is assumed that COVID-19 may be seasonal, but there are multiple other factors at play that can affect vitamin D levels that need to be considered.
-These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111].   
+While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.  
+Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.
```

Comment:
From my understanding, a seasonal disease is something different than a disease with seasonal severity. What exactly is the assumption of this review?

Reply:
This line refers to seasonal incidence (more infection around January etc.), in line with the hypothesis that vitamin D is a potential factor.
From my understanding, there has been little discussion on potential seasonal severity (the human impact of the virus e.g. increased hospitalisation levels etc.); Although, viral mutation patterns are being continually monitored.